src/network/messages/RequestReplay.cpp

```diff
@@ -22,4 +22,10 @@ namespace spy::network::messages {
     bool RequestReplay::operator==(const RequestReplay &rhs) const {
         return isEqual(rhs);
     }
+
+    bool RequestReplay::validate(RoleEnum playerRole, bool isReplayAvailable) const {
+        return playerRole != spy::network::RoleEnum::INVALID &&
+               playerRole != spy::network::RoleEnum::AI &&
+               isReplayAvailable;
```

Comment:
Die Nachricht sollte auch valide sein wenn der Server kein replay unterstützt, der client kann das ja nicht wissen

Reply:
Doch, der Client weiß das. Wird ihm in der StatisticsMessage mitgeteilt.